src/lib/MrDocsCompilationDatabase.cpp

```diff
@@ -507,18 +514,40 @@ adjustCommandLine(
     while (idx < cmdline.size())
     {
         // Parse one argument as a Clang option
-        // ParseOneArg updates Index to the next argument to be parsed.
-        unsigned const idx0 = idx;
         std::unique_ptr<llvm::opt::Arg> arg =
             opts_table.ParseOneArg(args, idx, visibility);
         if (!isValidMrDocsOption(workingDir, arg))
         {
             continue;
         }
-        new_cmdline.insert(
-            new_cmdline.end(),
-            cmdline.begin() + idx0,
-            cmdline.begin() + idx);
+
+        // Translate clang-cl /std: into clang -std=
```

Comment:
So `arg->render()` works for all kinds of arguments except `-std` (not that bad, but it would be good to include a small comment)? Or does that mean `arg->render()` requires custom logic for lots of kinds of arguments, but the only one for which we got custom logic is `-std` (much worse)?